enarx-keep-sgx-shim/src/main.rs

```diff
@@ -33,30 +86,18 @@ pub unsafe extern "C" fn entry(
     _r9: u64,
 ) -> ! {
     extern "C" {
-        fn do_syscall(
-            rdi: u64,
-            rsi: u64,
-            rdx: u64,
-            r10: u64,
-            r8: u64,
-            r9: u64,
-            rax: SysCall,
-        ) -> u64;
+        fn jump(rsp: u64, fnc: u64) -> !;
+        fn exit(code: u8) -> !;
```

Comment:
Where is this `code` used in `start.S`? I see:
```
exit:
    mov  $60, %rax
    syscall
    ud2
```

Reply:
@lkatalin Because of the calling convention, the `code` parameter is in `%rdi` when the function is called. `%rdi` is also the input to the `syscall`. So it is transparently passed into the `syscall` instruction.